base/doc/ltnews41.tex

```diff
@@ -186,13 +188,125 @@ \section{Replacement for the legacy mark mechanism}
+
+\item[\hook{build/column/before}, \hook{build/column/after}]
+   These two hooks enable packages to prepend or append code to
+   the column processing in the output routine. They are
+   implemented as mirrored hooks.
+
+   Technically, they are executed at the start and the end of the
+   internal \LaTeXe{} \cs{@makecol} command, respectively.  A
+   number of packages alter \cs{@makecol} to place code in exactly
+   these two places\Dash they can now simply add their code to the
+   hooks instead.
+\end{description}
+
+We also added a number of sockets for configuring the algorithm and to
+support tagging. One socket that is of interest for class files but
+also for user in the document preamble is
+\socket{build/column/outputbox}. It defines how the column text, the
+column floats (top and bottom) and the footnotes are combined,
+i.e. their order and spacing. To change the layout all one has to do
+is to assign a different predefined plug to the socket with
+\begin{flushleft}
+  \verb= \AssignSocketPlug{build/column/outputbox}=
+  \verb=                  {=\meta{plug-name}\verb=}=
+\end{flushleft}
+The predeclared plugs are the following:
+\begin{description}
+\item[\plug{space-footnotes-floats}]
+
+   After the galley text there is a vertical \cs{vfill}
+   followed by the footnotes, followed by the bottom floats, if any.
+
+\item[\plug{floats-footnotes-space}]
+
+   As before but the \cs{vfill} is at the bottom (the page is
+   ragged  bottom).
+
+\item[\plug{footnotes-space-floats}]
+
+   As before but the \cs{vfill} is between footnotes and floats.
+
+\item[\plug{space-floats-footnotes}]
+
+   Here the footnotes come last.\footnote{There are two more
+   permutations, but neither of them has ever been requested so
+   they aren't set up by default --- doing that in a class
+   would be trivial though.}
+
+\item[\plug{floats-footnotes}]
+
+   All excess space is distributed across the existing
+   glue on the page, e.g., within the text galley, the
+   separation between blocks, etc.
+   The order is text, floats, footnotes.
+
+\item[\plug{footnotes-floats}]
+
+   As the previous one but floats and footnotes are swapped. This is
+   the \LaTeX{} default for newer documents, i.e., this plug is
+   assigned to the socket when \cs{DocumentMetadata} is used.
+
+\item[\plug{footnotes-floats-legacy}]
+
+   As the previous one but \LaTeX{}'s bottom skip bug is not
+   corrected, i.e., in ragged bottom designs where footnotes
+   are supposed to be directly attached to the text, they suddenly
+   appear at the bottom of the page when the page is ended with
+   \cs{newpage} or \cs{clearpage}.
+   While this is clearly a bug, it was the case since the days
+   of \LaTeX~2.09; thus for
+   compatibility we continue to support this behavior.
+\end{description}
+There are more configuration possibilities, mainly for class
+developers; more documentation on those can be found in
+\cite[\S54 ltoutput.dtx]{41:source2e}.
+
+
```

Comment:
Not necessssarily now but at some point we'll probably need to put this kind of information in `clsguide` it will be hard to find in ltnews after a few releases.

Reply:
I agree, at least to the level that I described it in ltnews, the cite here is really to the more nitty gritty details, e.g., how to make yet another plug (which 0% people will ever do).  I could what I have placed into ltnews also place into clsguide